website/content/docs/connect/ca/vault.mdx

```diff
@@ -181,124 +170,207 @@ The configuration options are listed below.
-}
+  ```hcl
+  path "/sys/mounts/<root_pki_path>" {
+    capabilities = [ "read" ]
+  }
 
-path "/connect_root/root/sign-intermediate" {
-  capabilities = [ "update" ]
-}
+  path "/sys/mounts/<intermediate_pki_path>" {
+    capabilities = [ "read" ]
+  }
 
-path "/connect_inter/*" {
-  capabilities = [ "create", "read", "update", "delete", "list" ]
-}
+  path "/sys/mounts/<intermediate_pki_path>/tune" {
+    capabilities = [ "update" ]
+  }
+  ```
 
-path "auth/token/renew-self" {
-	capabilities = [ "update" ]
-}
+  </CodeBlockConfig>
 
-path "auth/token/lookup-self" {
-	capabilities = [ "read" ]
-}
-```
+1. Add the following rules to your Vault policy file so that Consul
+   has read-only access to the root PKI engine, can automatically rotate
+   intermediate CAs as needed, and has full use of the intermediate PKI engine:
 
-</CodeBlockConfig>
+  <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
-### Consul Managed PKI Paths
+  ```hcl
+  path "/<root_pki_path>/" {
+    capabilities = [ "read" ]
+  }
+  
+  path "/<root_pki_path>/root/sign-intermediate" {
+    capabilities = [ "update" ]
+  }
 
-The following Vault policy allows Consul to create and manage the PKI paths in Vault.
-Consul is granted the ability to create the PKI mount points and given full
-control of the Root and Intermediate or Leaf CA for Connect clients.
+  path "/<intermediate_pki_path>/*" {
+    capabilities = [ "create", "read", "update", "delete", "list" ]
+  }
+  ```
+  </CodeBlockConfig>
 
-In this example the `RootPKIPath` is `connect_root` and the `IntermediatePKIPath`
-is `connect_inter`. These values should be updated for your environment.
+1. Add the following rules to your Vault policy file so that Consul
+   can renew its Vault token if the token is renewable.
+   The rule enables the token to be renewed whether it is provided
+   [directly](#token) in the CA provider configuration or presented in an [auth method](#authmethod).
 
-<CodeBlockConfig filename="consul-managed-pki-policy.hcl">
+  <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
-```hcl
-# Consul Managed PKI Mounts
-path "/sys/mounts" {
-  capabilities = [ "read" ]
-}
+  ```hcl
+  path "auth/token/renew-self" {
+    capabilities = [ "update" ]
+  }
 
-path "/sys/mounts/connect_root" {
-  capabilities = [ "create", "read", "update", "delete", "list" ]
-}
+  path "auth/token/lookup-self" {
+    capabilities = [ "read" ]
+  }
+  ```
+  
+  </CodeBlockConfig>
+
+1. Conditional and temporary: Only include the following rules in your Vault policy file
+   while you are changing the CA provider from Vault to a different CA provider (such as Consul's built-in provider)
+   or changing the `RootPKIPath` used by the Vault CA provider.
+   Those CA provider modifications trigger a root CA change that requires an
+   extremely privileged root cross-sign operation.
+   We recommend removing this elevated privilege immediately after the CA provider modification is finished.
```

Comment:
```suggestion
1. You should only include the following rules in your Vault policy file
   while changing the CA provider from Vault to a different CA provider, such as Consul's built-in provider,
   or changing the `RootPKIPath` used by the Vault CA provider.
   The CA provider modifications trigger a root CA change that requires a
   highly-privileged root cross-sign operation.
   Remove the elevated privilege immediately after the CA provider modification is finished.
```
I get why you wanted to label the step, but it adds too many words without as much benefit. As far as the recommendation to remove the elevated privileges, should that be a final step at the end of this procedure? It would make this step shorter and provide a dedicated action for readers following the process.

Reply:
The policy snippets in steps 1-3 must **always** be in place for Consul to work.

This policy snippet is ideally only in place for a very brief period of time while performing a specific operation.

These four "steps" aren't really a _procedure_ to follow. I don't think it makes sense to add a "step 5" for removing the elevated privileges.

Maybe the right solution is to move this temporarily privilege into its own section? It's shared between Vault and Consul managed paths, so I don't need to duplicate it in both sections... I'll try that out so we can take a look.